test/tags.js

```diff
@@ -113,6 +114,19 @@ describe('fetchTags', () => {
     expect(await fetchTags({ ...options, startingDate: '2000-05-01' })).to.have.lengthOf(1)
   })
 
+  // with --sort-tags, we shouldn't sort by semver, keep git command's order.
+  it('supports --sort-tags', async () => {
```

Comment:
Can you include a test for “semver”, and also for something specific that is an actual use case? iow, “something” isn’t a thing, and in this test, tags are sorted by semver, so I’m a bit confused.

Reply:
What should I use instead of 'something'? For this test, it's important to keep command's order and it's not important what sortTags options is so I thunk I shouldn't use some actual one. 

> in this test, tags are sorted by semver
OK, I'll change to random order.

> Can you include a test for “semver”
I'll add it.

Reply:
What should I use instead of 'something'? For this test, it's important to keep command's order and it's not important what sortTags options is so I thunk I shouldn't use some actual one. 

> in this test, tags are sorted by semver

OK, I'll change to random order.

> Can you include a test for “semver”

I'll add it.

Reply:
The entire purpose of this flag is presumably to pick a specific non-semver ordering; let’s add a test dor that.

Reply:
Yes. The purpose is to pick a specific non-semver ordering but non-semver orders are depending on git so I think what I can test is keeping order from  command. Or Should I simulate some imaginary order with mock?

Reply:
Is there nothing in this project's git history that would work for an alternate sorting that's not semver?